sdk/healthinsights/azure-health-insights-clinicalmatching/src/main/java/com/azure/health/insights/clinicalmatching/AzureHealthInsightsServiceVersion.java

```diff
@@ -1,34 +1,40 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-// Code generated by Microsoft (R) AutoRest Code Generator.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
 
 package com.azure.health.insights.clinicalmatching;
 
 import com.azure.core.util.ServiceVersion;
 
-/** Service version of ClinicalMatchingClient. */
-public enum ClinicalMatchingServiceVersion implements ServiceVersion {
-    /** Enum value 2023-03-01-preview. */
+/**
+ * Service version of AzureHealthInsightsClient.
+ */
+public enum AzureHealthInsightsServiceVersion implements ServiceVersion {
```

Comment:
Same here - this type name should not be changed.

Reply:
Will change in a follow-up PR